NON_CONTAINERIZED_DEV_README.md

```diff
@@ -1,49 +1,101 @@
-(via `brew install rbenv` if on macOS) then `rbenv init` to set up rbenv in your shell. Then close out terminal and reopen 
-and run `rbenv install 3.1.2`. (version is current as of 08/09/2022)
-2. Run `bundler -v` to ensure Bundler is installed.  If not, `gem install bundler`.
-3. Run `node -v` to ensure Node is installed. If not, install via https://nodejs.org/en/download/
-4. Run `yarn -v` to ensure Yarn is installed. If not, install via `brew install yarn`
-5. `cd` to where you have the `single_cell_portal_core` Git repo checked out.
-6. Run `bundle install`
-7. Run `yarn install`
-8. Run `./rails_local_setup.rb` to will write out required variables into an shell env file (using your Broad username 
+```
+/* A buffer */
+typedef struct bson_buffer* bson_buffer_t;
+/* A position in the buffer */
+typedef int bson_buffer_position;
+
+/***** THE FOLLOWING IS THE LINE YOU NEED TO INSERT ****/
+int bson_buffer_get_max_size(bson_buffer_t buffer); 
+
+ /* Allocate and return a new buffer.
+ * Return NULL on allocation failure. */
+bson_buffer_t bson_buffer_new(void);
+```
+  -
+    - Run `pwd`, confirm you are still in `/Users/$(whoami)/.rbenv/versions/3.1.3/lib/ruby/gems/3.1.0/gems/bson_ext-1.5.1/ext/cbson`
+    - Run `make`, which will output warnings you can ignore
+    - Run `cd ../..`, i.e., go to the `bson_ext-1.5.1` directory
+    - Run `gem spec ../../cache/bson_ext-1.5.1.gem --ruby > ../../specifications/bson_ext-1.5.1.gemspec`
+    - Run `gem list bson_ext`, and confirm it outputs:
+
+```
+*** LOCAL GEMS ***
+bson_ext (1.5.1)
+```
+  -
+    - Change directories back to `single_cell_portal_core`
+10. Run `yarn install`
+11. Run `./rails_local_setup.rb` to will write out required variables into an shell env file (using your Broad username 
 to determine which `vault` paths to read from).
-9. Run the source command the script outputs -- this will export those needed variables into the current shell
-10. Add `config/local_ssl/localhost.crt` to your system's trusted certificates. On macOS, you can drag this file into the 
-keychain access app, use the "System" keychain, and the "Certificates" category. Then you will likely need to open the 
-newly added certificate in the keychain access app and update the "Trust" setting to be "Always Trust" rather than "Use 
-System Defaults".
-11. Run `rails s`
-12. If you're developing JS, for hot module replacement and live reload, in a separate terminal, run `bin/vite dev`
-13. If you are working on functionality that involves delayed jobs, like uploading data:
+12. Run the source command the script outputs -- this will export those needed variables into the current shell
+13. Add `config/certs/localhost.crt` to your system's trusted certificates. 
+  -  Automatic route (preferred), run `sudo security add-trusted-cert -d -r trustAsRoot -k /Library/Keychains/System.keychain config/certs/localhost.crt`
```

Comment:
Blocker: Please update -r trustAsRoot to -r trustRoot
Both options are mentioned at https://apple.stackexchange.com/questions/80623/import-certificates-into-the-system-keychain-via-the-command-line but only trustRoot worked

Reply:
Good catch.  I [copied that from a Slack comment](https://broadinstitute.slack.com/archives/CBEHTH601/p1678203008450679?thread_ts=1677782368.756229&cid=CBEHTH601).  Could you edit that comment so folks who stumble onto it use the working argument, and I'll plan to edit this doc as part of the PR that's opened in this repo?